package/network/config/ltq-vdsl-app/files/dsl_control

```diff
@@ -194,34 +195,39 @@ start_service() {
 
 	eval "xtse=\"\${xtse_xdsl_$annex}\""
 
-	[ -z "${xfer_mode}" ] && xfer_mode=ptm
```

Comment:
I've updated the pullrequest/commit because i found a missed change in the dsl_control init script, which seemed to got lost when i ported this change from lede-17.01 to master.

Without this change the auto mode doesn't really work, because it always assumed ptm/vdsl should be used.
Sorry.